src/ring_theory/ideal/basic.lean

```diff
@@ -590,6 +591,18 @@ not_is_field_iff_exists_ideal_bot_lt_and_lt_top.trans
     ⟨p, bot_lt_iff_ne_bot.mp (lt_of_lt_of_le bot_lt le_p), hp.is_prime⟩,
    λ ⟨p, ne_bot, prime⟩, ⟨p, bot_lt_iff_ne_bot.mpr ne_bot, lt_top_iff_ne_top.mpr prime.1⟩⟩
 
```

Comment:
Can you add a docstring here noting that `ideal.is_simple_order` also exists, and trades a requirement for commutativity for a requirement for a constructive inverse (or similar).

I think this raises that probably we're missing an `is_division_ring` predicate to generalize here.

Reply:
I added a docstring. Not sure if it aligns with what you are thinking.